src/workerd/api/tests/crypto-streams-test.js

```diff
@@ -107,6 +108,16 @@ export const digeststream = {
       deepStrictEqual(digest, check);
     }
 
+    {
+      const check = new Uint8Array([176, 224, 34, 147]);
+      const digestStream = new crypto.DigestStream('crc32');
+      const writer = digestStream.getWriter();
+      await writer.write(new Uint32Array([1, 2, 3]));
```

Comment:
This is fine but I'd like to see a test with multiple large writes.

Reply:
Is this blocking or a nit?

Reply:
Not blocking but something that would be very helpful

Reply:
Added one, we could add more later if you have more test cases in mind there.